website/cue/reference/components/sources/journald.cue

```diff
@@ -79,6 +79,34 @@ components: sources: journald: {
 				}
 			}
 		}
+		exclude_matches: {
+			common:      true
+			description: "The list of Journal field/value pairs to exclude from monitoring. When used with `exclude_units`, it will be merged into include_matches."
```

Comment:
I think there is something off about this. Did you really mean the exclude list is merged into an include list? Also, I don't think "Journal" needs to be capitalized, as it is not a proper name.
```suggestion
			description: "This list contains sets of field/value pairs that, if any are present in a journal entry, will cause the entry to be excluded from this source. If `exclude_units` is specified, it will be merged into this list."
```

Reply:
Thanks!
fixed e6068ee